client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -371,196 +399,274 @@ export default function CampaignItemDetails( props: Props ) {
+										{ ! isLoading && duration_days ? (
+											sprintf(
+												/* translators: %s is the duration in days */
+												_n( '%s day', '%s days', duration_days ),
+												formatNumber( duration_days, true )
+											)
+										) : (
+											<FlexibleSkeleton />
+										) }
+									</span>
 								</div>
-								<div className="campaign-item-details__main-stats-row">
-									<div>
-										<span className="campaign-item-details__label">
-											{ translate( 'Duration' ) }
-										</span>
-										<span className="campaign-item-details__text wp-brand-font">
-											{ ! isLoading ? durationFormatted : <FlexibleSkeleton /> }
-										</span>
-										<span className="campaign-item-details__details">
-											{ ! isLoading ? (
-												`${ duration_days } ${ translate( 'days' ) }`
-											) : (
-												<FlexibleSkeleton />
-											) }
-										</span>
-									</div>
-									<div>
-										<span className="campaign-item-details__label">{ translate( 'Budget' ) }</span>
-										<span className="campaign-item-details__text wp-brand-font">
-											{ ! isLoading ? totalBudgetFormatted : <FlexibleSkeleton /> }
-										</span>
-										<span className="campaign-item-details__details">
-											{ ! isLoading ? totalBudgetLeftFormatted : <FlexibleSkeleton /> }
-										</span>
-									</div>
-									<div>
-										<span className="campaign-item-details__label">
-											{ translate( 'Overall spending ' ) }
-										</span>
-										<span className="campaign-item-details__text wp-brand-font">
-											{ ! isLoading ? overallSpendingFormatted : <FlexibleSkeleton /> }
-										</span>
-									</div>
+								<div>
+									<span className="campaign-item-details__label">
+										{ translate( 'Active for' ) }
+									</span>
+									<span className="campaign-item-details__text wp-brand-font">
+										{ ! isLoading ? (
+											sprintf(
+												/* translators: %s is the number of active days */
+												_n( '%s day', '%s days', activeDays ),
+												formatNumber( activeDays, true )
+											)
+										) : (
+											<FlexibleSkeleton />
+										) }
+									</span>
+									<span className="campaign-item-details__details">
+										{ ! isLoading ? daysLeftformatted : <FlexibleSkeleton /> }
+									</span>
+								</div>
+								<div>
+									<span className="campaign-item-details__label">
+										{ translate( 'Spent so far' ) }
+									</span>
+									<span className="campaign-item-details__text wp-brand-font">
+										{ ! isLoading ? overallSpendingFormatted : <FlexibleSkeleton /> }
+									</span>
+									<span className="campaign-item-details__details">
+										{ ! isLoading ? (
+											/* translators: overallSpendingPercentage is the percentage of the total budget used */
+											translate( '%(overallSpendingPercentage) of total budget', {
```

Comment:
The replace placeholder here is not correct; you need to specify the type of the variable like this:
%s => String
%d => Double

Similar to the `sprintf` in C or PHP.

![Screenshot 2024-02-23 at 10 59 49](https://github.com/Automattic/wp-calypso/assets/1258162/f56c8637-f624-48d0-a19d-c9fc2e444f62)